### PR TITLE
Fixes type import in QASM parser

### PR DESCRIPTION
**Pull request by bryano**

### CODE REVIEW

File: cirq/contrib/qasm_import/_parser.py [hunk 1]
```diff
@@ -22,6 +23,10 @@
 from cirq.contrib.qasm_import._lexer import QasmLexer
 from cirq.contrib.qasm_import.exception import QasmException
 
+if TYPE_CHECKING:
+    # pylint: disable=unused-import
+    from typing import Tuple
```
Comment (Strilanc) [inline cirq/contrib/qasm_import/_parser.py]: This is for the line

`for qubits in op_qubits:  # type: Tuple[cirq.Qid]`

?

That refers to the type of the loop variable? You should instead change the `type: np.broadcast` on the previous line. For example:

`op_qubits = cast(Sequence[Sequence[ops.Qid]], functools.reduce(np.broadcast, args))`

Comment (bryano) [reply cirq/contrib/qasm_import/_parser.py]: I didn't even look at how `Tuple` was used. I'm just trying to shut pylint up...

Comment (bryano) [reply cirq/contrib/qasm_import/_parser.py]: Done

**Comment by bryano** [comment]
Is there a reason this file imports `cirq` at the top level rather than its submodules?

File: cirq/contrib/qasm_import/_parser.py (hunk 1 above)
Comment (balopat) [reply cirq/contrib/qasm_import/_parser.py]: I'm wondering why pylint is reporting this as an error for you and not on Travis / or on my machine. 
Is it the right version? `dev_tools/conf/pip-list-dev-tools.txt` declares `pylint==2.3.*`

**Comment by balopat** [comment]
> Is there a reason this file imports `cirq` at the top level rather than its submodules?

not really, it is simply an overlook on my end :)

File: cirq/contrib/qasm_import/_parser.py (hunk 1 above)
Comment (bryano) [reply cirq/contrib/qasm_import/_parser.py]: I have pylint 2.3.1

**Comment by Strilanc** [comment]
Oh sorry @bryano , I confused you with @balopat , which is why I was asking for a bunch of other stuff.

**Comment by bryano** [comment]
Would be good to know why pylint's complaining on my machine and not others.

**Comment by Strilanc** [comment]
Try `pip install --upgrade ./dev_tools/conf/pip-list-dev-tools.txt` and see if it persists.

**Comment by bryano** [comment]
No difference (all those requirements were already satisfied).

**Comment by balopat** [comment]
hmm, interesting, maybe it's something else. 
on my mac I have this setup: 
```
$ pylint --version
pylint 2.3.1
astroid 2.1.0
Python 3.6.4 (v3.6.4:d48ecebad5, Dec 18 2017, 21:07:28) 
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)]
```
What is your `pylint --version` output?

**Comment by vtomole** [comment]
@balopat We are aware of that warning: https://github.com/quantumlib/Cirq/issues/1701

**Comment by bryano** [comment]
Here's my pylint version info:
```
pylint 2.3.1
astroid 2.1.0
Python 3.6.8 |Anaconda, Inc.| (default, Dec 30 2018, 01:22:34) 
[GCC 7.3.0]
```

**Comment by bryano** [comment]
What surprises me is not that pylint complained on my machine, but that it didn't on Travis. When not type checking, that was an actually unused import.

**Pull request merged**